tracer/test/snapshots/AspNetMvc5TestsModuleOnly.VirtualApp.Integrated.__path=_badrequest_statusCode=500.verified.txt

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet.request,
+    Resource: GET /my-app/badrequest,
```

Comment:
FYI for other reviewers, the differences between this snapshot and the non-VirtualApp snapshot are the following:
- Resource has the additional `my-app/` prefix (which is incorrect but is being addressed by #2852)
- Service has the `/my-app` site appended
- http.url tag has the full URL which includes the `/my-app` prefix

Reply:
@andrewlock is it possible to configure the site name and the path separately? If so, I think it would be good to update these tests with such an IIS configuration (a separate PR would be fine)

Reply:
> is it possible to configure the site name and the path separately

I don't believe so. The site name _is_ the path. In the IIS UI you only specify the alias